### PR TITLE
Use PnP aware Yarn in CI.

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

**Review by stefanpenner — APPROVED** [review]
Thanks for the fix.
It also seems like the test passes on my machine with 10.9, and fails with 11.9. Could be my 10.9 environment is hosed?

**Comment by stefanpenner** [comment]
I suspect the yarn version in windows is old, let me bump it

File: appveyor.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ environment:
 
 install:
   - ps: Install-Product node $env:nodejs_version
+  - choco install yarn --version=1.13.0
```
Comment (stefanpenner) [inline appveyor.yml:15]: let's see if this works

**Comment by stefanpenner** [comment]
Ok that got us 1 step closer

```
'require(resolve-package-path)(ember-source-channel-url,
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: Invalid or unexpected token
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Object.<anonymous> ([eval]-wrapper:6:22)
    at Module._compile (module.js:653:30)
    at evalScript (bootstrap_node.js:479:27)
    at startup (bootstrap_node.js:180:9)
    at bootstrap_node.js:625:3
error Command failed with exit code 1.
$ node -r ./.pnp.js -e 'require("resolve-package-path")("ember-source-channel-url", __dirname)'
```

windows escaping FTL, going to move that test into its own file.

**Comment by stefanpenner** [comment]
released as v1.0.3 🎉

**Comment by davecombs** [comment]
🎉 !

**Comment by rwjblue** [comment]
@stefanpenner the reason Node 10 worked without this change, but Node 11 required it is due to a change (probably a bug?) in how Node 11 parses `NODE_OPTIONS`.

See:

* https://github.com/nodejs/node/issues/26521
* https://github.com/yarnpkg/yarn/issues/7092

**Comment by stefanpenner** [comment]
Looks like node folks fixed this in `11.12.0`

**Pull request merged**